units/XNA0303/XNA0303_unit.bp

```diff
@@ -319,14 +320,14 @@ UnitBlueprint {
                 },
             },
             RackFireTogether = false,
-            RackRecoilDistance = -1,
```

Comment:
This is a visual change that doesnt affect aiming or firing, its a weapon recoil thing, so doesnt need changing